cli/view.js

```diff
@@ -24,6 +24,7 @@ const addParser = (parser) => {
   subparser.addArgument('--handlers', {action: "store", metavar: "JS", help: "Overwrite the provided server handlers for client requests. See documentation for more details."});
   subparser.addArgument('--datasetDir', {metavar: "PATH", help: "Directory where datasets (JSONs) are sourced. This is ignored if you define custom handlers."});
   subparser.addArgument('--narrativeDir', {metavar: "PATH", help: "Directory where narratives (Markdown files) are sourced. This is ignored if you define custom handlers."});
+  subparser.addArgument('--customBuildOnly', {action: "storeTrue", help: "Error if a custom build is not found."});
```

Comment:
I like the intention here but the word "custom" is a bit overloaded here -- this is checking if an auspice build is present in the current working directory, not whether the build has been customised.

Reply:
I've split the addition of this argument into its own commit: 00902657df8e4391d039613712462c81fa6cc655

Ok I understand the difference now. I chose `--customBuildOnly` following the previous existence of `--customBuild`, which seemed to be doing something similar? I can't think of a better name without being verbose like `--onlyUseLocalAuspiceBuild`. Do you have suggestions?